packages/dashboard-frontend/src/contexts/WorkspaceActions/DeleteWarning/index.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import {
+  Button,
+  ButtonVariant,
+  Modal,
+  ModalVariant,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import React from 'react';
+
+import { WantDelete } from '@/contexts/WorkspaceActions';
+
+export type Props = {
+  isOpen: boolean;
+  wantDelete: WantDelete;
+  onClose: () => void;
+};
+
+export class WorkspaceActionsDeleteWarning extends React.PureComponent<Props> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isConfirmed: false,
+    };
+  }
+
+  private handleClose(): void {
+    this.props.onClose();
+  }
+
+  public render(): React.ReactElement {
+    const { isOpen, wantDelete } = this.props;
+
+    let warningMessage: React.ReactElement;
+    if (wantDelete.length === 1) {
+      const workspaceName = wantDelete[0];
+      warningMessage = (
+        <>
+          <b>{workspaceName}</b> workspace has <b>Per-user</b> storage type. You need to stop other
```

Comment:
I think we need to clarify in the warning what is the Per-user type e.g. one common PVC is used for all workspaces and that PVC has the RWO access mode https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes

Reply:
I have fixed it.